deb/openmediavault/usr/share/php/openmediavault/controlpanel/controlpanelabstract.inc

```diff
@@ -145,10 +173,12 @@ EOF;
 				"extjs6/classic/theme-triton/theme-triton-debug.js"
 			], $jsList);
 		} else {
-			$jsList = array_merge([
-				"extjs6/ext-all.js",
-				"extjs6/classic/theme-triton/theme-triton.js"
-			], $jsList);
+			if ($this->isLocaleRTL()) {
```

Comment:
The whole block is incorrect. These JS files must appear at the head of the list. You also forgot the OMV_DEBUG_EXTJS case. Please adapt according to this:
```
if (TRUE === \OMV\Environment::getBoolean("OMV_DEBUG_EXTJS", FALSE)) {
			if ($this->isLocaleRTL()) {
				array_unshift($jsList, "extjs6/ext-all-rtl-debug.js");
			} else {
				array_unshift($jsList, "extjs6/ext-all-debug.js");
			}
			array_unshift($jsList, "extjs6/classic/theme-triton/theme-triton-debug.js");
		} else {
			if ($this->isLocaleRTL()) {
				array_unshift($jsList, "extjs6/ext-all-rtl.js");
			} else {
				array_unshift($jsList, "extjs6/ext-all.js");
			}
			array_unshift($jsList, "extjs6/classic/theme-triton/theme-triton.js");
		}
		foreach ($jsList as $jsListv) {
			print "<script type='application/javascript' src='{$jsListv}'></script>\n";
		}
```